app/models/goal.py

```diff
@@ -1,5 +1,14 @@
 from app import db
 
-
+# One goal has many tasks
 class Goal(db.Model):
     goal_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+
+    tasks = db.relationship('Task', back_populates="goal")
+
+    def to_dict(self):
+        return {
+            "id": self.goal_id,
+            "title": self.title
+        }
```

Comment:
Great `to_dict()` helper function! 

Remember to always put a newline at the end of the file -- this is good style practice in Python and a requirement in some other languages.